plugin-management-library/src/test/java/io/jenkins/tools/pluginmanager/impl/PluginManagerIntegrationTest.java

```diff
@@ -298,7 +298,7 @@ public void verifyDownloads_smoke() throws Exception {
 
         // Second cycle, with plugin update and new plugin installation
         Plugin trileadAPI = new Plugin("trilead-api", "1.0.13", null, null);
-        Plugin snakeYamlAPI = new Plugin("snakeyaml-api", "1.27.0", null, null);
+        Plugin snakeYamlAPI = new Plugin("snakeyaml-api", "1.31-84.ve43da_fb_49d0b", null, null);
```

Comment:
I think this setting can be retained to continue testing that behavior remains compatible even for older releases.
```suggestion
        Plugin snakeYamlAPI = new Plugin("snakeyaml-api", "1.27.0", null, null);
```

Reply:
I think fine to update some firewalls may get angry at vulnerable deps getting downloaded although could also be left